Metadata structure simple (#390)
FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec: specification/metadata/focusversion.md

> # FOCUS Version

Reviewer: We also need a DataGeneratorVersion (or something like that) to account for data generators creating multiple versions of their support of a specific FOCUS version (e.g., with new custom columns).

Author: Added DataGeneratorVersion

> The declared FOCUS version of the FOCUS data within the schema.

Reviewer 2: ```suggestion
The version of FOCUS utilized for building the dataset.
```

Reviewer 2: Here I think we need to be mindful that it's not necessarily conveying compliance. I assume many providers will likely not be 100% compliant but will provide data starting with a particular FOCUS version and incrementally add support.

Reviewer 3: Yeah that is why I have this as a metadata signifying the version declared in that schema. If this is saying that they are following a specific version in general, the problem is that how does the practitioner know the version for older data. That is why I have it attached to the schema itself.

The FOCUSVersion MUST be provided in the metadata. FOCUSVersion MUST be of type String and MUST NOT contain null values. FOCUSVersion MUST match one of the published versions of the FOCUS specification. FOCUSVersion MUST match the version of the FOCUS specification that the FOCUS dataset conforms to.

## Metadata ID

> FOCUSVersion

Reviewer 4: FocusVersion?

Same rationale as for SkuId, SkuPriceId

Reviewer: +1

Author: Updated
